verticaltabs.js

```diff
@@ -92,37 +100,31 @@ VerticalTabs.prototype = {
       });
       sidetabsbutton.style.MozAppearance = 'none';
       sidetabsbutton.style.setProperty('-moz-image-region', 'rect(0, 16px, 16px, 0)', 'important');
+      let boundCheckbrighttext = checkbrighttext.bind(sidetabsbutton);
```

Comment:
I think, instead of calling bind, we can (should) use arrow functions, like so…
``` javascript
let checkbrighttext = () => {
  ...
};
checkbrighttext();
window.addEventListener('customizationchange', checkbrighttext);
```
both because we never use the unbound version, and to keep the namespace cleaner. 🙂